src/main/java/org/sonatype/nexus/ci/config/Nxrm3Configuration.groovy

```diff
@@ -69,5 +70,36 @@ class Nxrm3Configuration
         error(e, 'Nexus Repository Manager 3.x connection failed')
       }
     }
+
+    @Override
+    FormValidation doCheckServerUrl(@QueryParameter String value) {
+      def validation = super.doCheckServerUrl(value)
+
+      if (validation.kind != OK) {
+        return validation
+      }
+
+      // check nexus version, warn if < 3.13.0 PRO
+      try {
+        def response = new XmlSlurper().parseText(new URL("${value}/service/rest/wonderland/status").text)
+        def edition = response.edition.text()
+        def version = response.version.text()
+        def (major, minor) = version.tokenize('.').take(2).collect { it as int }
+
+        if (!edition.equalsIgnoreCase('pro') || major < 3 || minor < 13) {
+          return FormValidation.warning(
+              "NXRM ${edition} ${version} found. Some operations require a Professional Edition Nexus Repository " +
+                  "Manager server of version 3.13.0 or newer; use of an incompatible server will result in failed " +
```

Comment:
Suggestion: "Professional Edition Nexus Repository Manager server" reads weirdly to me, I think mostly because that's not how I hear it, not because it's not legit.  Suggest instead "Nexus Repository Manager server Professional Edition".  Also, +0 on "Edition", I could go either way.  I mostly just say "PRO".

Reply:
On the other hand, specifying "Professional" first highlights the important point that this warning is trying to make which is Pro is required.

Reply:
Why guess? When confronted with how to phrase something about the product, who do we talk to? Presumably this is something we want to be somewhat consistent about.

Reply:
> specifying "Professional" first highlights the important point

@nmcafee I did consider this but IMO it's only important in 1/2 the messages.  The other you're using pro so the version is important.
FYI, I also did a search through sonatype.com for wording to see if I was crazy and there's no mention of Edition anywhere on the site I saw, except in reference to Trial.  That is phrased as I suggested "Nexus Repository Manager Trial Edition".
But anyway I see it both ways, thus "suggestion".

> When confronted with how to phrase something about the product, who do we talk to?

@kellyrob99 I would have said you ;)  Maybe you have a suggestion who to ask?
Respectfully, I think we are inconsistent even on our website and docs (some places say Pro, some places have the mid branding "Nexus Repository Manager" inferrring Pro) so my guess is the answer will vary based on who you ask, so despite the winky who is the final source is a legit Q.

Reply:
@joedragons This is PO+Marketing largely (and has definitely not been consistent in the past) so I'd start there